tests/test_unclassified_job.py

```diff
@@ -26,3 +25,13 @@ def test_open_unclassified_failure_log(base_url, selenium):
     treeherder_page.open_next_unclassified_failure()
     logviewer_page = treeherder_page.job_details.open_logviewer()
     assert logviewer_page.is_job_status_visible
+
+
+@pytest.mark.nondestructive
+def test_view_unclassified_jobs(base_url, selenium):
+    page = TreeherderPage(selenium, base_url).open()
+    assert len(page.result_sets[0].testfailed_job_title) >= 0
```

Comment:
This appears to be checking that the title for the first testfailed job for the first result set is non-zero. The test name suggests that we're testing the unclassified jobs filter, and I'm not sure this assertion is appropriate. Wouldn't this fail if the first result set didn't have any test failures? Also, we already have a jobs property - could we use that to return the title, and filter it based on visibility or status?